Dockerfile

```diff
@@ -1,16 +1,23 @@
-FROM node:16
-
+# Build
+FROM node:16.14.0-slim AS build
 WORKDIR /app
 
 COPY package*.json ./
-
-RUN npm ci
+RUN npm install --production
```

Comment:
Remove this production from the build step, otherwise, your npm build will fail (since it wouldn't have types!!)